policyengine_canada/parameters/gov/age_exemption.yaml

```diff
@@ -6,5 +6,5 @@ metadata:
   name: income_tax_max_age
   label: Income tax age exemption
   reference:
-  - title: Title
-    href: https://example.com
\ No newline at end of file
+  - title: Government of Canada - 9.2.4 Tax brackets and rates
```

Comment:
Move to `parameters/gov/cra/tax/income/exempt/age.yaml`

Also specify if this is inclusive of 65 or not